src/Application/Regulation/Query/GetRegulationOrdersToDatexFormatQueryHandler.php

```diff
@@ -49,7 +49,13 @@ public function __invoke(GetRegulationOrdersToDatexFormatQuery $query): array
             $vehicleConditions = [];
 
             foreach ($row['restrictedVehicleTypes'] ?: [] as $restrictedVehicleType) {
-                $vehicleConditions[] = new DatexVehicleConditionView($restrictedVehicleType);
+                $vehicleConditions[] = new DatexVehicleConditionView(
+                    $restrictedVehicleType,
```

Comment:
```suggestion
                    vehicleType: $restrictedVehicleType,
```

Comme tu le spécifies sur les autres, je le ferais aussi ici.